server/tilesource/base.py

```diff
@@ -1108,6 +1169,24 @@ def tileIteratorAtAnotherScale(self, sourceRegion, sourceScale=None,
                                          targetScale, targetUnits)
         return self.tileIterator(region=region, scale=targetScale, **kwargs)
 
+    def getSingleTile(self, *args, **kwargs):
+        """
+        Return the first tile from an iterator.  This takes exactly the same
+        parameters as tileIterator.
```

Comment:
Update the documentation to say it can be used to get any specific tile (not just the first tile).